decidim_app-design/app/assets/stylesheets/_steps.scss

```diff
@@ -0,0 +1,45 @@
+.steps{
```

Comment:
Maybe this naming is easy to confuse with a participatory process' steps?

Reply:
Renamed to `.wizard__steps`